jellyseer/umbrel-app.yml

```diff
@@ -0,0 +1,34 @@
+manifestVersion: 1
+id: jellyseerr
+category: media
+name: Jellyseerr
+version: "1.7.0"
+tagline: Beautiful media discovery for Jellyfin users
+description: >-
+  Jellyseerr is a request management and media discovery tool built to work with your existing Jellyfin ecosystem.
+
+
+  Jellyseerr scans your Jellyfin libraries at regular intervals, so it knows which items are already available on your server.
+  It also integrates with the popular DVR applications Radarr and Sonarr, and supports activity monitoring within Jellyseerr itself.
+
+
+  🛠️ SET-UP INSTRUCTIONS
+
+  During initial set-up, you will need to input your Umbrel device's IP address to connect to Jellyfin (and optional services such as Radarr and Sonarr).
+  You can find your device's IP address by visiting your router's admin dashboard or by using an IP scanning tool like Angry IP Scanner.
+developer: Fallenbagel
+website: https://github.com/Fallenbagel/jellyseer
+dependencies: []
+repo: https://github.com/Fallenbagel/jellyseer
+support: https://github.com/Fallenbagel/jellyseer/issues
+port: 5055
```

Comment:
This field is what determines what port an app will be accessible at on Umbrel. So to avoid clashing with overseerr, we can change this to a free port. `5056` is free!